cmd/ddev/cmd/testdata/TestCmdAddonGetEnvironmentFlag/busybox/docker-compose.busybox.yaml

```diff
@@ -0,0 +1,22 @@
+services:
+  busybox:
+    container_name: ddev-${DDEV_SITENAME}-busybox
+    image: busybox:${DDEV_BUSYBOX_VERSION:-stable}
+    command: tail -f /dev/null
+    restart: "no"
+    # These labels ensure this service is discoverable by ddev.
+    labels:
+      com.ddev.site-name: ${DDEV_SITENAME}
+      com.ddev.approot: $DDEV_APPROOT
+    environment:
+      # This variable cannot be overridden with env_file, because it's already declared here
+      - THIS_VARIABLE_CANNOT_BE_CHANGED_FROM_ENV=true
+      # This variable can never be overridden from the .env.busybox.example file
+      - BUSYBOX_EXAMPLE_VARIABLE=${BUSYBOX_EXAMPLE_VARIABLE:-notset}
+      # DDEV_REDIS_VERSION from .env.redis can be passed here during the 'docker-compose config' step,
+      # when all .env.* files are passed to 'docker-compose config'
+      - CUSTOM_REDIS_VERSION=${DDEV_REDIS_VERSION:-notset}
```

Comment:
It's a chicken situation when I pass all the `.env.*` files to `docker-compose config` because I can't know what services I have until that config is complete, but at the time it is running, the variable `DDEV_REDIS_VERSION` is already expanded, and I can't do nothing about it.

This is the first thing I don't like about this PR.